src/lib/db/feature-toggle-client-store.ts

```diff
@@ -45,16 +45,20 @@ export default class FeatureToggleClientStore
         r: any,
         includeId: boolean = true,
     ): IStrategyConfig {
-        const strategy = {
-            name: r.strategy_name,
-            constraints: r.constraints || [],
-            parameters: r.parameters,
-            id: r.strategy_id,
-        };
-        if (!includeId) {
-            delete strategy.id;
+        if (includeId) {
+            return {
+                name: r.strategy_name,
+                constraints: r.constraints || [],
+                parameters: r.parameters,
+                id: r.strategy_id,
```

Comment:
are there any reasons why we should not deliver strategy config id to client SDKs?